src/views/Timesheets/Timesheets.module.scss

```diff
@@ -1,3 +1,11 @@
+.tooltip {
+  background-color: var(--mui-palette-neutral-contrastText);
+  color: rgba(255, 255, 255, 0.87);
+  box-shadow: 0 3px 6px #00000029, 0 3px 6px rgba(0, 0, 0, 0.23);
```

Comment:
why is the shadow a coded color?

Reply:
This was the original coded color that was in the in-line style and we decided not to change it.